Source/MBF.SystemCore.pas

```diff
@@ -241,13 +254,18 @@ procedure TSystemCore.ConfigureTimer;
   TicksPerMillisecond := GetSysTickClockFrequency div 1000;
 
 {$if defined(CPUARM)}
+
```

Comment:
Is there a way that this can get moved to SystemCore.SAMCD as it is SAMCD specific?

Reply:
For sure. I do not see any problems with that.